code/index.html

```diff
@@ -2,21 +2,103 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <title>World of AI</title>
     <link rel="stylesheet" href="./style.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <!-- dont forget the meta tag for responsiveness -->
   </head>
   <body>
     <header>
       <!-- Some kind of logo and navbar -->
+
+      <img src="./logo.svg" alt="logo">
+
+      <p>WORLD OF AI</p>
+
+      <nav>
+        <a href="#">Latest technology</a>
+        <a href="#">Machine learning</a>
+        <a href="#">Robotics</a>
+        <a href="#">Human vs AI</a>
+      </nav>
+
+      
+      <div class="hamburger">
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
+     
+
     </header>
 
     <section class="big-news">
       <!-- The big news content -->
+      <img src="./neon code.jpg" alt="neon code">
+      <div>
+        <h2>Welcome to the World of AI</h2>
+        <p>All you need to know about the latest AI technology. From machine learning to artificial consciousness. The future is already here, are you ready to enter?</p>
+      </div>
+
     </section>
 
-    <section class="">
+      <hr>
+
+    <section class="news-container">
       <!-- The other news  -->
+
+      <div class="flex-child">
+        <img src="./robot.jpg" alt="robot">
+        <h3>The uncanny valley effect and its consequences</h3>
+      </div> 
+
+      <div class="flex-child">
+        <img src="./ready or not.jpg" alt="sign ready or not">
+        <h3>How prepared are you for AI? Sed egestas egestas</h3>
+        
+        </div>
+
+      <div class="flex-child">
+          <img src="./truck.jpg" alt="truck">
+          <h3>Self-driving trucks, now more common than old-style trucks</h3>
+          </div>
+    
+
+      <div class="flex-child">
+        <img src="./nasa.jpg" alt="inside a spaceship">
+        <h3>First android on Mars next year   Mauris vitae ultricies id absp</h3>
+        </div>  
+
+      
+      <div class="flex-child">
+      <img src="./face recognition.jpg" alt="woman face recognition">
+      <h3>Your face is your new credit card Hac habitasse est</h3>
+      </div>
+
+      
+
+      <div class="flex-child"> 
+      <img src="./brain.jpg"  alt="brain">
+      <h3>Connect your brain to AI, pros and cons magna eget est</h3>
+      </div>
+
+      <div class="flex-child">
+      <img src="./neon header.jpg" alt="neon lights">
+      <h3>Does artificial intelligence give life a new meaning?</h3>
+      </div> 
+        
+      </section>
     </section>
+
+    
+
+    <footer>
+      <h3>Wanna learn more about AI?</h3>
+      <p>Contact us for more information</p>
+      <p>AI@org.com</p>
+      <div>
+        <a href="https://www.elementsofai.se/"><button>Go to Elements of AI - free online course</button></a>
+        </div>
+    </footer>
```

Comment:
Really nice that you had time to add a footer, and nice that you added an actual link for the button 🎉
Here the indentation is almost perfect, just the `</div>` that should be a bit more to the left.